### PR TITLE
Rework description of using the app with recent changes

### PR DESCRIPTION
**Pull request by rohanjr**
changelog_begin
changelog_end

### Pull Request Checklist

- [ ] Read and understand the [contribution guidelines](https://github.com/digital-asset/daml/blob/master/CONTRIBUTING.md)
- [ ] Include appropriate tests
- [ ] Set a descriptive title and thorough description
- [ ] Add a reference to the [issue this PR will solve](https://github.com/digital-asset/daml/issues), if appropriate
- [ ] Include changelog additions in one or more commit message bodies between the `CHANGELOG_BEGIN` and `CHANGELOG_END` tags
- [ ] Normal production system change, include purpose of change in description

NOTE: CI is not automatically run on non-members pull-requests for security
reasons. The reviewer will have to comment with `/AzurePipelines run` to
trigger the build.

### CODE REVIEW

File: docs/source/getting-started/index.rst [hunk 1]
```diff
@@ -91,23 +91,29 @@ The command should automatically open a window in your default browser at http:/
...
-This should give you a idea of the app's functionality.
-
-Next, we'll look at the :doc:`architecture of our app <app-architecture>` and the individual components.
+Now you should see the login page for the social network.
+For simplicity of this app, there is no password required to sign up or log in.
+To learn how to handle authentication, see this blog post about `DAML and Auth0 <https://blog.daml.com/daml-driven/easy-authentication-for-your-distributed-app-with-daml-and-auth0>`_ or the :doc:`full documentation </app-dev/authentication>`.
+
+First enter your name and click "Sign up".
+You should see the main screen with two panels for your friends and the entire network.
+Initially these are both empty as you don't have friends yet!
```
Comment (cocreature) [inline docs/source/getting-started/index.rst]: rude!

File: docs/source/getting-started/index.rst [hunk 2]
```diff
@@ -91,23 +91,29 @@ The command should automatically open a window in your default browser at http:/
...
+Go ahead and add some using the text box and add button.
+
+You'll notice that the newly added friends are under your name in the Friends panel but don't appear in the network panel.
```
Comment (cocreature) [inline docs/source/getting-started/index.rst]:
```suggestion
You'll notice that the newly added friends are under your name in the *Friends* panel but don't appear in the *Network* panel.
```

File: docs/source/getting-started/index.rst [hunk 3]
```diff
@@ -91,23 +91,29 @@ The command should automatically open a window in your default browser at http:/
...
+The latter is because 1. they have not signed up and are not parties on the ledger and 2. they have not yet added you as a friend.
```
Comment (cocreature) [inline docs/source/getting-started/index.rst]: inline lists are hard to read imho, maybe split it up over multiple lines?

File: docs/source/getting-started/index.rst [hunk 4]
```diff
@@ -91,23 +91,29 @@ The command should automatically open a window in your default browser at http:/
...
+In our social network, friendships can go in a single direction.
+By adding a friend, say Alice, your User contract becomes visible to Alice but not vice versa.
```
Comment (cocreature) [inline docs/source/getting-started/index.rst]:
```suggestion
By adding a friend, say Alice, your ``User`` contract becomes visible to Alice but not vice versa.
```

File: docs/source/getting-started/index.rst [hunk 5]
```diff
@@ -91,23 +91,29 @@ The command should automatically open a window in your default browser at http:/
...
+We will see how we encode this model in DAML in the next section.
+
+To make your friendships reciprocal, open a new browser window to the same http://localhost:3000 address.
```
Comment (cocreature) [inline docs/source/getting-started/index.rst]:
```suggestion
To make your friendships reciprocal, open a new browser window at http://localhost:3000.
```

File: docs/source/getting-started/index.rst [hunk 6]
```diff
@@ -91,23 +91,29 @@ The command should automatically open a window in your default browser at http:/
...
+(Having separate windows allows you to see both you and your friend's screens together.)
+Once you sign up as your friend Alice, you'll notice your name in her network.
+When Alice finally adds you back as a friend, you can see her in your network as well.
+(Just open the previous window for this - no need to reload the page!).
```
Comment (cocreature) [inline docs/source/getting-started/index.rst]: It’s hard to keep track of which windows you are referriing to here. Maybe something like “open the previous window where you logged in with your own name”.

File: docs/source/getting-started/index.rst (hunk 4 above)
Comment (rohanjr) [reply docs/source/getting-started/index.rst]: Realised I haven't talked about contracts at this point, I'll reword it.

File: docs/source/getting-started/index.rst (hunk 3 above)
Comment (rohanjr) [reply docs/source/getting-started/index.rst]: I might change this later, but I don't want to over emphasise it with a roomy list.

**Pull request merged**